### PR TITLE
fix: swap text for stars/forks in repo page OG image

### PR DESCRIPTION
**Pull request by brandonroberts**
## Description

- Flips the text for stars/forks on the open graph image

## Mobile & Desktop Screenshots/Recordings

Before:

image

After:

## Steps to QA

1. View OpenGraph image from deploy preview for repo page
2. Note Stars/Forks text matches image

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [x] Tier 4

### CODE REVIEW

File: netlify/edge-functions/repo-page-card.tsx [hunk 1]
```diff
@@ -179,7 +179,7 @@ export default async function handler(req: Request) {
                 <span style={statValueContainerStyles}>
                   <span style={statValueStyles}>{humanizeNumber(forkCount)}</span>
                 </span>
-                <span style={statSubTextStyles}>Stars</span>
+                <span style={statSubTextStyles}>Forks</span>
```
Comment (nickytonline) [inline netlify/edge-functions/repo-page-card.tsx:182]: oof. how'd I miss that. 🙈

**Review by nickytonline — APPROVED** [review]
🚢

File: netlify/edge-functions/repo-page-card.tsx (hunk 1 above)
Comment (brandonroberts) [reply netlify/edge-functions/repo-page-card.tsx:182]: 😅  I was about to share it and noticed it

**Review by bdougie — APPROVED** [review]

File: netlify/edge-functions/repo-page-card.tsx (hunk 1 above)
Comment (nickytonline) [reply netlify/edge-functions/repo-page-card.tsx:182]: 😅

**Pull request merged**